File: Modul3 Latihan/Latihan1.py
stack = []

stack.append("aku")
stack.append("anak")
stack.append("Indonesia")

print("next!" + stack[-1])
stack.append("Raya")
print(stack.pop())

count = stack.count("Aku")
while count > 0:
    stack.pop()
    count -= 1

print(stack.pop())
print(not stack)

File: Modul3 Latihan/src/Modul3/Latihan1.java
package Modul3;

import java.util.Stack;

public class Latihan1 {


    public static void main(String[] args) {

        Stack st = new Stack();

        st.push("aku");
        st.push("anak");
        st.push("Indonesia");


        System.out.println("next!" +st.peek());
        st.push("Raya");
        System.out.println(st.pop());

        int count= st.search("Aku");
        while (count!= -1 && count > 1) {
            st.pop();
            count--;
        }
        System.out.println(st.pop());
        System.out.println(st.empty());
    }
}